unstructured/partition/pdf.py

```diff
@@ -314,19 +314,27 @@ def _process_pdfminer_pages(
                 if _text.strip():
                     text_segments.append(_text)
                     element = element_from_text(_text)
-                    element._coordinate_system = PixelSpace(
+                    coordinate_system = PixelSpace(
                         width=width,
                         height=height,
                     )
-                    element.coordinates = ((x1, y1), (x1, y2), (x2, y2), (x2, y1))
-                    element.metadata = metadata
+                    points = ((x1, y1), (x1, y2), (x2, y2), (x2, y1))
+                    coordinates_metadata = CoordinatesMetadata(
+                        points=points,
+                        coordinate_system=coordinate_system,
+                    )
+                    element.metadata = ElementMetadata(
+                        filename=filename,
+                        page_number=i + 1,
+                        coordinates=coordinates_metadata,
+                    )
                     page_elements.append(element)
 
         sorted_page_elements = sorted(
             page_elements,
             key=lambda el: (
-                el.coordinates[0][1] if el.coordinates else float("inf"),
-                el.coordinates[0][0] if el.coordinates else float("inf"),
+                el.metadata.coordinates.points[0][1] if el.metadata.coordinates else float("inf"),
+                el.metadata.coordinates.points[0][0] if el.metadata.coordinates else float("inf"),
```

Comment:
@qued , what is the index of the bottom right point? thinking we can add negative values for those Y and X points as well.

Reply:
Should be index 2 (so 3rd in order).